ext/JSON3Ext.jl

```diff
@@ -0,0 +1,54 @@
+module JSON3Ext
+
+if isdefined(Base, :get_extension)
+    using JSON3
+else
+    using ..JSON3
+end
+
+using Dagger
+
+function logs_to_chrome_trace(logs::Dict)
+    execution_logs = Dict{Int,Any}()
+    Dagger.logs_event_pairs(logs) do w, start_idx, finish_idx
+        category = logs[w][:core][start_idx].category
+        if category == :compute
+            tid = logs[w][:id][start_idx].thunk_id # thunk id
+            if !haskey(execution_logs, tid)
+                execution_logs[tid] = Dict{Symbol,Any}()
+            end
+            t_start = logs[w][:core][start_idx].timestamp / 1e3 # us
+            t_stop = logs[w][:core][finish_idx].timestamp / 1e3 # us
+            proc = logs[w][:id][start_idx].processor
+            execution_logs[tid][:ts] = t_start
+            execution_logs[tid][:dur] = t_stop - t_start
+            execution_logs[tid][:pid] = proc.owner
+            execution_logs[tid][:tid] = proc.tid # thread id
```

Comment:
One thing to beware of - only `ThreadProc` has `tid`; the GPU processors (like `CuArrayDeviceProc`) does not have this field. Similarly to how we have `Dagger.root_worker_id(::Processor)` to get the Distributed worker that a processor is associated with (essentially the `owner` field), we could also have a helper like `Dagger.processor_thread_id(::Processor)` which would return `tid` for `ThreadProc`, and something like `device_id` for GPU processors.

You are welcome to ignore this or put it off for later - I will also happily add it when I get around to it. If you want to do the easy thing, you can also just skip any events where `!(proc isa Dagger.ThreadProc)`, and maybe just emit a warning that events were skipped? It's totally up to you!

Reply:
For now I'll put all the non-`ThreadProc` as a single "unknown" pid/tid and print warning. I haven't used Dagger with GPU yet, but I plan to, so maybe I'll add a revision then